src/EnergyPlus/CMakeLists.txt

```diff
@@ -784,6 +784,9 @@ if(BUILD_GROUND_PLOT)
 endif()
 if(UNIX AND NOT APPLE)
   target_link_libraries( energypluslib dl )
+  if (CMAKE_COMPILER_IS_GNUCXX AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 9.0)
+    target_link_libraries( energypluslib util )
+  endif()
```

Comment:
Wouldn't link without this on GCC 10. Undefined reference to `forkpty` and `openpty`

Reply:
@Myoldmopar